src/js/queries/getOfficialDocumentsCollectionDocumentsQuery.js

```diff
@@ -0,0 +1,25 @@
+const getOfficialDocumentsCollectionDocumentsQuery = `
+  query getOfficialDocumentsCollectionDocumentsQuery($id: ID) {
+    officialDocumentCollectionDocuments(officialDocumentCollection: $id, orderBy: "-page__date") {
```

Comment:
I am not sure if we should paginate this query. It does not seem to time out now, but.....

Reply:
What's cool is that we have the option to paginate it, where we didn't before on the old version of official docs. Let's make an issue to paginate it next sprint. 215 pages in a single query is still a lot. It's fine for now if it's not timing out.

Reply:
issue #4619